app/views/layout.slim

```diff
@@ -8,9 +8,9 @@ html
 	body
 		- if pundit_user
 			nav
-				h1: a href="/" Tint
+				a href="/" Tint
 				- if site
-					section
+					header
```

Comment:
Does this even belong in the nav element?  There are no links here at all... not a big deal, just a thought I had